makefile

```
@@ -61,18 +74,39 @@ gen-proto:
 
 	cp -R internal/proto/github.com/open-telemetry/opamp-go/protobufs/* protobufs/
 	rm -rf internal/proto/github.com/
+	$(MAKE) fmt
 
 .PHONY: gomoddownload
 gomoddownload:
-	go mod download
+	$(GOCMD) mod download
 
 .PHONY: install-tools
-install-tools:
-	cd $(TOOLS_MOD_DIR) && go install github.com/ory/go-acc
+install-tools: $(TOOLS_BIN_NAMES)
+
+$(TOOLS_BIN_DIR):
+	mkdir -p $@
+
+$(TOOLS_BIN_NAMES): $(TOOLS_BIN_DIR) $(TOOLS_MOD_DIR)/go.mod
+	cd $(TOOLS_MOD_DIR) && $(GOCMD) build -o $@ -trimpath $(filter %/$(notdir $@),$(TOOLS_PKG_NAMES))
 
 .PHONY: tidy
 tidy:
 	rm -fr go.sum
-	go mod tidy
-	cd internal/examples && rm -fr go.sum && go mod tidy
-	cd internal/tools && rm -fr go.sum && go mod tidy
+	$(GOCMD) mod tidy
+	cd internal/examples && rm -fr go.sum && $(GOCMD) mod tidy
+	cd $(TOOLS_MOD_DIR) && rm -fr go.sum && $(GOCMD) mod tidy
+
+.PHONY: fmt
+fmt: common/gofmt common/goimports common/gofumpt
+
+.PHONY: common/gofmt
```

Unclear why this uses `common/` prefix in target names. I suggest we simplify and remove it.
```suggestion
.PHONY: gofmt
```

---

looks simpler, changed in https://github.com/open-telemetry/opamp-go/pull/356/commits/eed6e204d026cbba650126f92e3c0b978e5e2072